README.md

```diff
@@ -5,15 +5,15 @@ Flutter plugin for integrating Blueshift's iOS and Android SDKs to your Flutter
 ## Installation
 
 ```shell
-$ fluter pub get blueshift-flutter-plugin
+$ fluter pub get blueshift-plugin
 ```
 
 ## Android and iOS Integration
 
 Refer to the below documents to integrate the Blueshift SDK for Android and iOS.
-
-- [Android SDK Integration](./Android.md)
-- [iOS SDK Integration](./iOS.md)
+- [Flutter Plugin Integration](https://developer.blueshift.com/docs/install-and-set-up-flutter-plugin)
+- [Android SDK Integration](https://developer.blueshift.com/docs/integrate-your-flutter-android-app)
+- [iOS SDK Integration](https://developer.blueshift.com/docs/integrate-your-flutter-ios-app)
```

Comment:
I think we should not give the links to android and ios docs here. The user should get navigated to these when they visit the **Flutter Plugin Integration** doc